06-multiple-regression.Rmd

```diff
@@ -997,17 +997,22 @@ get_regression_table(model_2_parallel_slopes) %>%
 
 Observe how the regression table for the interaction model has 2 more rows (6 versus 4). This reflects the additional "complexity" of the interaction model over the parallel slopes model. 
 
-Furthermore, note in Table \@ref(tab:model2-interaction) how the *offsets for the slopes* `perc_disadvan:sizemedium` being 0.146 and `perc_disadvan:sizelarge` being 0.189 are small relative to the *slope for the baseline group* of small schools. In other words, all three slopes are similarly negative: $-2.932$ for small schools, $-2.786$ $(= -2.932 + 0.146)$ for medium schools, and $-2.743$ $(= -2.932 + 0.146)$ for large schools. These results are suggesting that irrespective of school size, the relationship between average math SAT scores and the percent of the student body that is economically disadvantaged is similar and, alas, quite negative.
+Furthermore, note in Table \@ref(tab:model2-interaction) how the *offsets for the slopes* `perc_disadvan:sizemedium` being 0.146 and `perc_disadvan:sizelarge` being 0.189 are small relative to the *slope for the baseline group* of small schools of $-2.932$. In other words, all three slopes are similarly negative: $-2.932$ for small schools, $-2.786$ $(=-2.932 + 0.146)$ for medium schools, and $-2.743$ $(=-2.932 + 0.146)$ for large schools. These results are suggesting that irrespective of school size, the relationship between average math SAT scores and the percent of the student body that is economically disadvantaged is similar and, alas, quite negative.
```

Comment:
This isn't quite right. This has that both -2.786 =-2.932 + 0.146 and -2.743 =-2.932 + 0.146. Is there a reason why these are still hard-coded? Seems simpler to just use R variables here instead?

Reply:
Fixed it. $-2.743$ $(=-2.932 + 0.189)$